app/server/tests/functions/postFunctions.js

```diff
@@ -0,0 +1,38 @@
+require('dotenv').config();
+const supertest = require('supertest');
+const endpoint = process.env.API_REF;
+const request = supertest(endpoint);
+
+
+function createPost(token){
+    return request
+        .post('/post')
+        .set({authorization: `Bearer ${token}`});
+} 
+
+function getAllPosts(token){
+    return request
+        .get(`/post`)
+        .set({authorization: `Bearer ${token}`});
+} 
+
+function getPost(id,token){
+    return request
+        .get(`/post/${id}`)
+        .set({authorization: `Bearer ${token}`});
+} 
+
+function editPost(newTitle,newDescript,token){
+    return request
+        .patch(`/post/${id}`)
+        .set({authorization: `Bearer ${token}`})
+        .send({'title': newTitle, 'description': newDescript,}); 
```

Comment:
Looks good. FYI in case you didn't realize: You can edit whichever fields you like except for creator. You just need to specify the ones you are editing in the body of the request, like you are here for title and description.

Reply:
I removed it. nice catch 👍